<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProDialog</class>
 <widget class="QDialog" name="AddProDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加新的程序</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ic.qrc">
    <normaloff>:/ic/f.png</normaloff>:/ic/f.png</iconset>
  </property>
  <widget class="QLineEdit" name="proNameEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="proPathEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>文件/程序名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>文件路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="commitButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancleButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ic.qrc"/>
 </resources>
 <connections/>
</ui>
